cli/command/service/rollback.go

```diff
@@ -0,0 +1,27 @@
+package service
+
+import (
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/spf13/cobra"
+)
+
+func newRollbackCommand(dockerCli *command.DockerCli) *cobra.Command {
+	options := newServiceOptions()
+
+	cmd := &cobra.Command{
+		Use:     "rollback [OPTIONS]",
+		Aliases: []string{"rollback"},
+		Short:   "Rollback a service",
+		Args:    cli.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			flags := cmd.Flags()
+			flags.Bool("rollback", true, "")
```

Comment:
Since the command is rollback, we don't need a rollback flag, right?

I guess the calling code expects this?  We should refactor `runUpdate()` so that we have a function we can call directly that doesn't require such a flag.